src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,96 @@
+/* = Navigation
+============================ */
+.tribe-common {
+
+	.tribe-common-c-nav {
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+
+		ul {
+			display: block;
+			list-style: none;
+			width: 100%;
+		}
+	}
+
+	.tribe-common-c-nav__prev,
+	.tribe-common-c-nav__next {
+		display: inline-flex;
```

Comment:
Let's also try to avoid `inline-flex` unless these elements absolutely need to be `inline`. Inline elements add a space below the element for text descenders that cannot be removed, and it makes spacing impossible to get perfect.